database/postgres/db.sql

```diff
@@ -2,26 +2,16 @@ GRANT ALL PRIVILEGES ON DATABASE "WSO2AM_DB" TO wso2carbon;
     \c "WSO2AM_DB"
         BEGIN TRANSACTION;
         
-        CREATE SEQUENCE SUBSCRIBER_SEQUENCE START WITH 1 INCREMENT BY 1;
-        CREATE TABLE IF NOT EXISTS SUBSCRIBER (
-        SUBSCRIBER_ID INTEGER DEFAULT nextval('subscriber_sequence'),
-        USER_ID VARCHAR(255) NOT NULL,
-        ORGANIZATION VARCHAR(100) NOT NULL,
-        EMAIL_ADDRESS VARCHAR(256) NULL,
-        DATE_SUBSCRIBED TIMESTAMP NOT NULL,
-        CREATED_BY VARCHAR(100),
-        CREATED_TIME TIMESTAMP,
-        UPDATED_BY VARCHAR(100),
-        UPDATED_TIME TIMESTAMP,
-        PRIMARY KEY (SUBSCRIBER_ID),
-        UNIQUE (ORGANIZATION,USER_ID)
+        CREATE TABLE IF NOT EXISTS APK_USER (
```

Comment:
do we need APK_ prepend here? other table doesn't have such thing, since APK name might change and will need migration if we go like this?

Reply:
USER is a reserved keyword in Postgresql. We might occur some other complications if we use USER.

Reply:
maybe internal_user for name? let's ask and confirm from team otherwise no need

Reply:
Changed from d08893f88d81476a12b4d8f6d77ae14a7a0e72e5